verifier/detail_template.html

```diff
@@ -388,7 +388,7 @@
                         } else {
                             output = item[key];
                         }
-                        html.push('<td onclick=' + onclick_call +'>' + output +'</td>');
+                        html.push('<td onclick=' + onclick_call +'>' + escapeHtml(output) +'</td>');
```

Comment:
Thought: We should ideally be building this up using a proper templating engine or else as a proper DOM tree so that we can write to `innerText` properties and the escaping is done automatically for us; it is bad practice to build up HTML as a string. But, that's cleanup we should do later, outside of the scope of this PR.